tf2x.yml

```diff
@@ -0,0 +1,15 @@
+name: tf2.10X
+channels:
+  - defaults
+  - conda-forge
+  - anaconda
+dependencies:
+  - python=3.9
+  - cudnn
+  - nibabel
+  - gputil
+  - ants
```

Comment:
I see that you have omitted my channel `pnlbwh`. Who is providing `ants` now then?
https://github.com/pnlbwh/CNN-Diffusion-MRIBrain-Segmentation/blob/master/environment_gpu.yml

Reply:
conda-forge